.changeset/quick-streets-trade.md

```
@@ -0,0 +1,9 @@
+---
+"starlight-view-modes": minor
+---
+
+Adds support for Astro v6, drops support for Astro v5.
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)